src/TLuaInterpreter.cpp

```diff
@@ -14768,8 +14768,8 @@ void TLuaInterpreter::setAtcpTable(const QString& var, const QString& arg)
 {
     lua_State* L = pGlobalLua;
     lua_getglobal(L, "atcp"); //defined in LuaGlobal.lua
-    lua_pushstring(L, var.toLatin1().data());
-    lua_pushstring(L, arg.toLatin1().data());
+    lua_pushstring(L, var.toUtf8().constData());
+    lua_pushstring(L, arg.toUtf8().constData());
```

Comment:
ATCP has largely been replaced (as I understand it) by GMCP - and I am not sure it supports anything other than ASCII - I dunno whether that matters here.